Pinta.Effects/Effects/WarpEffect.cs

```diff
@@ -50,12 +50,17 @@ public WarpEffect ()
 
 	public override void LaunchConfiguration ()
 	{
-		EffectHelper.LaunchSimpleEffectDialog (this);
+		Chrome.LaunchSimpleEffectDialog (this);
 	}
 
 	protected double DefaultRadius { get; private set; } = 0;
 	protected double DefaultRadius2 { get; private set; } = 0;
 
+	protected abstract LivePreviewManager LivePreview { get; }
```

Comment:
This seems questionable since there isn't actually an interface to be mocked out. I'd suggest leaving the live preview manager for a future PR that introduces an interface (and then enables a unit test for Polar Inversion)

Reply:
Understood. I've now left it out